src/server/world.zig

```diff
@@ -30,7 +30,29 @@ pub const WorldSettings = struct {
 	gamemode: Gamemode = .creative,
 	allowCheats: bool = false,
 	testingMode: bool = false,
+	pub fn initFromZon(zon: ?ZonElement) WorldSettings {
+		var self: WorldSettings = undefined;
+		if(zon) |worldSettings| {
+			const gamemode: main.game.Gamemode = std.meta.stringToEnum(main.game.Gamemode, worldSettings.get([]const u8, "gamemode", "survival")) orelse blk: {
+				std.log.err("Invalid gamemode specified in launchConfig: {s}. Defaulting to survival.", .{worldSettings.get([]const u8, "gamemode", "survival")});
+				break :blk .survival;
+			};
+			self = .{
+				.gamemode = gamemode,
+				.testingMode = worldSettings.get(bool, "testingMode", false),
+				.allowCheats = worldSettings.get(bool, "allowCheats", false),
+			};
+		} else {
+			self = .{
+				.gamemode = .creative,
```

Comment:
Please use the same defaults everywhere.
Ideally just use the field defaults directly:
```zig
var result: WorldSettings = .{};
result = .{
	.testingMode = zon.get(bool, "testingMode", result.testingMode);
...
};
```